docs/hooks/index.md

```diff
@@ -35,64 +27,48 @@ An _Action_ defines one or more _Hooks_ to execute. lakeFS supports three types
-| name               | Identify the Action file                                  | String     | false    | If missing, filename is used instead                                    |
-| on                 | List of events that will trigger the hooks                | List       | true     |                                                                         |
-| on<event>.branches | Glob pattern list of branches that triggers the hooks     | List       | false    | **Not applicable to Tag events.** If empty, Action runs on all branches |
-| hooks              | List of hooks to be executed                              | List       | true     |                                                                         |
-| hook.id            | ID of the hook, must be unique within the action.         | String     | true     |                                                                         |
-| hook.type          | Type of the hook ([types](#hook-types))                   | String     | true     |                                                                         |
-| hook.description   | Optional description for the hook                         | String     | false    |                                                                         |
-| hook.if            | Expression that will be evaluated before execute the hook | String     | false    | No value is the same as evaluate `success()`                            |
-| hook.properties    | Hook's specific configuration                             | Dictionary | true     |                                                                         |
-
-Example:
+| `name               `| Identify the Action file                                  | String     | false    | If missing, filename is used instead                                    |
+| `on                 `| List of events that will trigger the hooks                | List       | true     |                                                                         |
+| `on<event>.branches `| Glob pattern list of branches that triggers the hooks     | List       | false    | **Not applicable to Tag events.** If empty, Action runs on all branches |
+| `hooks              `| List of hooks to be executed                              | List       | true     |                                                                         |
+| `hook.id            `| ID of the hook, must be unique within the action.         | String     | true     |                                                                         |
+| `hook.type          `| Type of the hook ([types](#hook-types))                   | String     | true     |                                                                         |
+| `hook.description   `| Optional description for the hook                         | String     | false    |                                                                         |
```

Comment:
Strange: We both say "optional" and that it's not required.

Reply:
thanks. Fixed.